src/legacy/core_plugins/data/public/search/index.ts

```diff
@@ -19,5 +19,5 @@
 
 export * from './aggs';
 export { getRequestInspectorStats, getResponseInspectorStats } from './utils';
-export { serializeAggConfig } from './expressions/utils';
+export { serializeAggConfig, deserializeAggConfig } from './expressions/utils';
```

Comment:
deserialize shouldn't be statically exported as it requires state (access to field formats registry)

Reply:
returned back importing of "deserializeAggConfig" as before